artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/ServerLocatorImpl.java

```diff
@@ -1298,6 +1298,18 @@ public ServerLocatorImpl setInitialMessagePacketSize(final int size) {
       return this;
    }
 
+   @Override
+   public int getOnMessageCloseTimeout() {
+      return config.onMessageCloseTimeout;
+   }
+
+   @Override
```

Comment:
The API docs specify the value set must be greater than zero, but there isn't any validation of that, should there be?

Reply:
Hmmm...That comment was left-over from a copy & paste. I went looking through the existing code to see how validation is done for other parameters, and I'm not actually seeing any. I'm continuing my investigation.

Reply:
I did look into what it does with that configuration value and since it is passing into a CountdownLatch is seems like it will not trigger an exception but will return immediately so from the standpoint of being able to be set to zero or negative it works, although seems an absurd value to assign.

Reply:
It would seem weird to configure those if actually knowing thats what it then does, but I'd guess some might think one of those would simply mean _keep waiting_ (e.g as is the described behaviour in the JMS spec, albeit that also originally described to effectively deadlock yourself as result hehe), especially if not reading the javadoc (should there also be actual-docs addition?).

Reply:
I changed the JavaDoc to be more clear. I didn't want to add validation as it wouldn't be consistent with the rest of the method and adding validation to all the places that could realistically use it is well beyond the scope of this PR.